### DISCUSSION TITLE
SkyWalking UI - Setting of listening IP address

### DISCUSSION THREAD

**Discussion by jarofi** [apache/skywalking#10952] [Ideas]
Hello everybody,

Now only the following information can be changed in the SkyWalking Native UI configuration file:
1. Listening port
2. Backend connect info

Would it be possible to extend the configuration options by adding **listening addresses** (for example as a comma separated list). My suggestion:
`serverAddress: ${SW_SERVER_ADDRESS:-http://localhost}`

Thank you in advance for your assessment.

**Comment by pg-yang** [comment]
You don't need to open a new discussion on this topic. And you could modify the source code, and build the project to achieve your aim.  Submitting PR to upstream also is welcome.

**Comment by jarofi** [comment]
@pg-yang I'm sorry, but I'm not a developer, so I can't prepare code extensions. I thought the purpose of the Ideas category is to collect suggestions/tips for Skywalking improvements/extensions from users. Please, what's the correct procedure? Thank you in advance for your reply.

**Reply by wu-sheng** [reply]
This is an open-source community. Currently, it is not supported. 
If you want to add, it has to be done by some developers. Otherwise, you have to stay what is being offered.

**Comment by jarofi** [comment]
I accept explanation by @wu-sheng
